Realm/RealmSwift/Migration.swift

```diff
@@ -54,23 +54,26 @@ public class Migration {
 
     private var rlmMigration: RLMMigration
 
-    init(_ rlmMigration: RLMMigration) {
+    internal init(_ rlmMigration: RLMMigration) {
         self.rlmMigration = rlmMigration
     }
 }
 
 public class MigrationObject : Object {
-    subscript(key: String) -> AnyObject? {
-        if (self.objectSchema[key].type == RLMPropertyType.Array) {
-            return listProperties[key]
+    public override subscript(key: String) -> AnyObject? {
```

Comment:
Did you clear build/DerivedData and run the MigrationExample after making these changes? Would be nice to know we didn't break anything. Would also be prudent to run all of the other examples.